src/resources/user/user.model.ts

```diff
@@ -77,6 +77,14 @@ userSchema.pre<UserDocument>('save', function(next) {
 	next()
 })
 
+userSchema.virtual('teams', {
+	ref: 'team',
+	localField: 'teamIds',
+	foreignField: '_id',
+	justOne: false,
+	options: {sort: {name: -1}, limit: 100},
```

Comment:
why we need the sort and limit here and imo we dont need the teams property here in user model